### ISSUE TITLE
Simple Task Editor hitbox on iOS is frustratingly difficult to tap

### ISSUE THREAD

**Issue by leotong** [standardnotes/forum#800]
Hi! The task editor works pretty well generally (on desktop) but on iOS it is incredibly frustrating trying to tick off list items. Sometimes it takes 20-30 finger taps to get it to mark it as completed.

![RPReplay_Final1575910134 2019-12-09 17_56_31](https://user-images.githubusercontent.com/304326/70455847-45be8680-1aad-11ea-8c9d-54afdb9d695e.gif)

**Comment by bryvin** [comment]
Hi @leotong ,

Sorry for your experience with this editor; we recently made a change to the behavior here due to another similar report. Unfortunately iOS keeps cached versions of the Web Editors until a new version of the actual SN App has been released. Since we have not yet made a new update to the SN App itself, the cache is likely still providing you the old copy of the editor. 

If you could try un-installing and re-installing the SN app (note this will clear any local data you have, so use this only if your account is synced with an SN account). After the re-install give the Task Editor a shot again and let us know if it's any easier for you.

If not we will continue to look further into this and make adjustments to help users.

**Comment by leotong** [comment]
@bryvin thanks! I'll give that a try later. Which version of STE is the latest? The one I have is 1.3.5

**Comment by bryvin** [comment]
The Simple Task Editor is indeed on 1.3.5. If you don't mind me asking how did you check the version you have installed from iOS?

In any case if you could still try the re-install of the mobile app when you get a chance to see if it still may be a caching issue that would be great. We are going to also look into providing a better way to clear the cache without having to release a new version of the mobile app.

Thanks for your help with this! We will look into correcting this behavior if the re-install doesn't fix it for you :)

**Comment by leotong** [comment]
I checked from the Mac app, where I uninstalled the app > closed the Mac app + closed the iOS app > reopened the Mac app, installed STE again > closed the Mac app > reopened the Mac app > and re-selected some STE on those notes which were previously STE task lists > reopened the iOS app and selected STE on some other notes which were previously STE task lists.

The behaviour seems to be similar and I can't really notice any improvement. P.S. I don't have fat fingers or anything... Perhaps making the checkbox/hitbox bigger overall might help? Thanks!
